src/coreclr/jit/llvm.cpp

```diff
@@ -1804,19 +1804,19 @@ void Llvm::storeObjAtAddress(Value* baseAddress, Value* data, StructDesc* struct
     }
 }
 
-void Llvm::buildStoreObj(GenTreeObj* storeOp)
+void Llvm::buildStoreBlk(GenTreeBlk* blockOp)
 {
-    if (!storeOp->OperIsBlk())
+    if (!blockOp->OperIsBlk())
     {
         failFunctionCompilation(); // cant get struct handle. TODO-LLVM: try as an assert
     }
```

Comment:
```suggestion
```
Dead code; both `GT_BLK` and `GT_BLK` are `OperIsBlk`.

Reply:
ok, thanks, thought you might comment on that.